tests/resource-custom-coverage-test.js

```diff
@@ -135,6 +135,10 @@ describeApplication('ResourceCustomCoverage', () => {
           it('removes the new row', () => {
             expect(ResourceCoverage.dateRangeRowList().length).to.equal(1);
           });
+
+          it('does not display the saving will remove message', () => {
```

Comment:
If this _was not_ previously displayed, we should use `it.always` to ensure it doesn't get displayed after this test gets run.